test/views/components/SummaryListSpec.scala

```diff
@@ -31,14 +31,17 @@ class SummaryListSpec extends SpecBase {
   "component" should {
 
     "display correct contents" when {
+
       "when change is false" in new Setup {
```

Comment:
Could you please remove 'when' from the description as its incorrect here? Its not necessary to address this comment as it is not part of your change. So resolve it, irrespective of change made or not